src/ResourceManager/AnalysisServices/Commands.AnalysisServices/Models/AnalysisServicesClient.cs

```diff
@@ -102,10 +106,31 @@ public AnalysisServicesClient()
                     updateParameters.BackupBlobContainerUri = backupBlobContainerUri;
                 }
 
+                if (ReadonlyReplicaCount != -1)
```

Comment:
When will this be -1?

Reply:
when the user does not provide read only replica count to the cmdlet in SET-SERVER